DUnitX.Assert.pas

```diff
@@ -1228,6 +1229,8 @@ class procedure Assert.DoesNotContain(const theString, subString: string; const
 class procedure Assert.EndsWith(const subString : string; const theString : string; const ignoreCase : boolean; const message : string);
 begin
   DoAssert;
+  if (subString = '') or (theString = '') then
+    raise ETestFailure.create('Strings empty !');
```

Comment:
Why is this directly raising ETestFailure rather than calling FailFmt? This will cause problems for some older versions of delphi (XE3 I think).